config/jobs/kubernetes/sig-network/sig-network-misc.yaml

```diff
@@ -265,6 +265,261 @@ presubmits:
       testgrid-alert-email: antonio.ojea.garcia@gmail.com, danwinship@redhat.com
       description: Uses kubetest to run e2e Conformance, SIG-Network tests against a cluster using nftables created with cluster/kube-up.sh
 
+  - name: pull-kubernetes-e2e-gce-kubedns-performance
```

Comment:
we don't need this on presubmit, we can skip it, just one that gives us signal is enough